src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -133,6 +137,23 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
       Failure(e)
   }
 
+  private def addAmdXml(datasetId: String, metadata: File): Try[Unit] = Try {
+    configuration.fedoraProvider.map {
+      provider =>
+        val foXml = provider.loadFoXml(datasetId).get
+        val amd = getAmd(foXml).get
+        (metadata / "amd.xml").writeText(amd.serialize)
+    }.getOrElse {
+      logger.warn(s"No amd.xml added, no fedora was configured")
+    }
+  }
+
+  private def copyMigrationFiles(metadata: File, migration: File, fromVault: Boolean): Try[Unit] = Try {
+    val migrationFiles = if (fromVault) Seq("provenance.xml", "amd.xml", "dataset.xml", "files.xml") else Seq("provenance.xml", "emd.xml", "dataset.xml", "files.xml")
```

Comment:
```suggestion
    val migrationFiles = Seq("provenance.xml", "dataset.xml", "files.xml", if (fromVault) "amd.xml" else "emd.xml", "dataset.xml")
```
perhaps extract var of the if-then-else.